client/src/pages/course/student/cross-check-review.tsx

```diff
@@ -117,10 +118,17 @@ function Page(props: CoursePageProps) {
     });
 
   useEffect(() => {
```

Comment:
you may use `useAsyncEffect`

Reply:
I would leave it as is - don't want to add new deps for that simple case